tools/divergent_links.py

```diff
@@ -0,0 +1,106 @@
+"""This script help checking divergent links.
+
+That is to say, links to the same page,
+that have different titles.
```

Comment:
PS. Please let's ditch my terminology. It's terrible. Nobody is going to be able to remember which side of divergent or convergent they are on. We should probable stick with names like same-name-different-URL links and same-URL-different-name links, unless we can think of something creative and memorable.

Reply:
Or maybe, less of a mouthful... name-consistent links (same name, different URLs) versus name-inconsistent links (different names, same URL)

Reply:
But I don't like that either because it makes it sound like one is bad (inconsistent) and the other good (consistent) whereas neither of them are good. So maybe, URL-inconsistent links versus name-inconsistent links.

Reply:
I've reworded.